### PR TITLE
docs: canonicalize the use of quotes for default values

### PR DESCRIPTION
**Pull request by geeknees**
#### Description

Notation distortion has been corrected.

#### Motivation and Context

Closes # https://github.com/starship/starship/issues/2935

#### How Has This Been Tested?

- [x] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: docs/config/README.md [hunk 1]
```diff
@@ -330,7 +330,7 @@ date is read from the `AWSUME_EXPIRATION` env var.
 
 | Option              | Default                                                           | Description                                                                                                 |
 | ------------------- | ----------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- |
-| `format`            | `'on [$symbol($profile )(\($region\) )(\[$duration\] )]($style)'` | The format for the module.                                                                                  |
+| `format`            | `"on [$symbol($profile )(\($region\) )(\[$duration\] )]($style)"` | The format for the module.                                                                                  |
```
Comment (davidkna) [inline docs/config/README.md:333]: `'` is used over `"` in some cases to avoid needing to escape `\` as `\\`. Please avoid replacing `'` with `"`.

Comment (geeknees) [reply docs/config/README.md:333]: Thank you very much. I see.I could modify it to `\\` with double quotes, but is it preferable to change it back to single quotes?

Comment (davidkna) [reply docs/config/README.md:333]: I would prefer keeping the single quotes to keep it more readable.

Comment (geeknees) [reply docs/config/README.md]: Corrected here. Please check it.
https://github.com/starship/starship/pull/4428/commits/2ef07e79b59dbc76704beec8aef45f503f344bb7

File: docs/config/README.md [hunk 2]
```diff
@@ -1683,7 +1683,7 @@ To enable it, set `disabled` to `false` in your configuration file.
 
 [git_metrics]
 added_style = "bold blue"
-format = '[+$added]($added_style)/[-$deleted]($deleted_style) '
+format = "[+$added]($added_style)/[-$deleted]($deleted_style)"
```
Comment (davidkna) [inline docs/config/README.md:1686]:
```suggestion
format = "[+$added]($added_style)/[-$deleted]($deleted_style) "
```

**Review by davidkna — CHANGES_REQUESTED** [review]

File: docs/config/README.md (hunk 2 above)
Comment (geeknees) [reply docs/config/README.md]: Thank you very much. Please let me know if I need to combine all the commits into one.

**Review by davidkna — APPROVED** [review]

**Comment by davidkna** [comment]
Thanks for the fixes!

**Pull request merged**